.changeset/pretty-lions-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"jspsych": minor
```

Comment:
shouldn't this just be a patch change?

Reply:
You are right. Fixed that in the latest commit.